include/ut.h

```diff
@@ -138,23 +135,44 @@ UT_test_t *UT_add_test( UT_test_suite_t *pSuite, const char *pTitle, UT_TestFunc
 #include <TestRun.h>
 #include <CUnit.h>
 
-#define UT_VERSION "1.0.0"
-
+/* UT test macros */
 #define UT_PASS(msg) CU_PASS(msg)
-#define UT_FAIL(msg) CU_FAIL(msg)
 
-/* Fatal test macros */
-#define UT_FAIL_FATAL(msg) CU_FAIL_FATAL(msg)
+#define UT_FAIL(msg) CU_FAIL_FATAL(msg)
+/* NON-FATAL equivalent of UT_FAIL test macro */
+#define UT_FAIL_NOT_FATAL(msg) CU_FAIL(msg)
+
 #define UT_ASSERT(value) CU_ASSERT_FATAL(msg)
+/* NON-FATAL equivalent of UT_ASSERT test macro */
+#define UT_ASSERT_NOT_FATAL(value) CU_ASSERT(msg)
```

Comment:
Duplicated the bug in UT_ASSERT_NOT_FATAL(value) -> CU_ASSERT(msg)

Where `msg` should be `value`

Reply:
Done